public/app/plugins/panel/alertlist/UnifiedalertList.test.tsx

```diff
@@ -222,4 +223,14 @@ describe('UnifiedAlertList', () => {
       expect.anything()
     );
   });
+
+  it('should render authorization error when user has no permission', async () => {
+    jest.spyOn(contextSrv, 'hasPermission').mockReturnValue(false);
+
+    await act(async () => {
```

Comment:
Are you able to do this without `act`, and instead do

```
expect(await screen.findByRole('alert', ...
```

below?

Reply:
Yes!